pages/work/jebbit.vue

```diff
@@ -0,0 +1,147 @@
+<template>
+  <div class="section flex flex-wrap justify-center">
+    <div class="max-w-2xl w-full">
+      <div class="p-4 lg:p-12">
+        <div class="text-xs">
+          Case Study
+        </div>
+
+        <h1 class="text-4xl lg:text-6xl">
+          Jebbit
+        </h1>
+
+        <p>
+          Jebbit is an enterprise software platform that enables a future of
+          total data transparency between brands and consumers. Jebbit’s mission
+          is to make every brand-and-consumer interaction assumption free by
+          asking consumers explicitly and directly about their interests,
+          motivations, and preferences.
+        </p>
+      </div>
+
+      <div class="bg-black flex flex-wrap h-64 md:h-140 justify-center items-center">
```

Comment:
The bg color behind the logo is intended to be the brand color, so we should add a `jebbit` color to tailwind and use `bg-jebbit` here. I think maybe `#97D9D8` but not sure what the official color is for Jebbit.